### PR TITLE
added pre-commit hook

### PR DESCRIPTION
**Pull request by PrimkY**
+ fix: added timer field to question
+ feat: added pre-commit hook

### CODE REVIEW

File: src/questions/questions.model.ts [hunk 1]
```diff
@@ -1,39 +1,87 @@
...
+  @ApiProperty({
+    example: 'that test will show us ur power in Node js',
+    description: 'make it by ur own / required',
+  })
   @Column({ type: DataType.STRING, allowNull: false })
   description: string;
 
-  @ApiProperty({example: '1', description: 'number of topic that should be import into question'})
+  @ApiProperty({
+    example: 'that test will show us ur power in Node js',
```
Comment (raman-davidovich) [inline src/questions/questions.model.ts:75]: Copy error. Replace with a record: "Time to resolve the question in seconds"

File: src/questions/questions.model.ts [hunk 2]
```diff
@@ -1,39 +1,87 @@
...
+    description: 'make it by ur own / required',
```
Comment (raman-davidovich) [inline src/questions/questions.model.ts:76]: Copy error. Replace with a record from create-question.dto.ts: "timer in seconds / required'"

File: src/quiz/quiz.controller.ts [hunk 3]
```diff
@@ -1,58 +1,59 @@
...
+import { Quiz_Question } from './quiz.question.model';
 
 @ApiTags('Quiz')
 @Controller('/api/quiz')
 export class QuizController {
   constructor(private readonly quizService: QuizService) {}
 
-  @ApiOperation({summary: 'Method to view all quizs'})
-  @ApiResponse({status: 200, type: Quiz})
+  @ApiOperation({ summary: 'Method to view all quizs' })
```
Comment (raman-davidovich) [inline src/quiz/quiz.controller.ts:24]: Typing error. Plural fom for quiz should be "quizzes"

File: src/quiz/quiz.model.ts [hunk 4]
```diff
@@ -1,30 +1,45 @@
...
+import { Question } from '../questions/questions.model';
+import { Quiz_Question } from './quiz.question.model';
+import { User } from '../user/user.model';
 
 interface QuizCreationAttrs {
   title: string;
 }
 
-@Table({tableName: 'quizs' , createdAt: false, updatedAt: false})
+@Table({ tableName: 'quizs', createdAt: false, updatedAt: false })
```
Comment (raman-davidovich) [inline src/quiz/quiz.model.ts:19]: Typing error. Plural fom for quiz should be "quizzes"

File: src/topic/dto/create-topic.dto.ts [hunk 5]
```diff
@@ -1,6 +1,9 @@
-import { ApiProperty } from "@nestjs/swagger";
+import { ApiProperty } from '@nestjs/swagger';
 
 export class CreateTopicDto {
-  @ApiProperty({example: 'NodeJS question', description: 'Simple sample for theme / required'})
+  @ApiProperty({
+    example: 'NodeJS question',
```
Comment (raman-davidovich) [inline src/topic/dto/create-topic.dto.ts:5]: I propose to leave just a name of the technology ("NodeJS").

File: src/topic/topic.controller.ts [hunk 6]
```diff
@@ -1,33 +1,39 @@
...
-  @ApiResponse({status: 200, type: [Topic]})
+  @ApiOperation({ summary: 'Method to get one topic by id' })
+  @ApiResponse({ status: 200, type: [Topic] })
   @Get('/:id')
   getById(@Param('id') id: number) {
-    return this.topicService.getById(id);
+    return this.topicService.getTopicById(id);
+  }
+
+  @ApiOperation({ summary: 'Method to delete question' })
```
Comment (raman-davidovich) [inline src/topic/topic.controller.ts:33]: Typing error. Should be replaced on @ApiOperation({ summary: 'Method to delete topic' })

File: src/topic/topic.model.ts [hunk 7]
```diff
@@ -1,20 +1,27 @@
...
+    unique: true,
+    autoIncrement: true,
+    primaryKey: true,
+  })
   id: number;
 
-  @ApiProperty({example: 'NodeJS question', description: 'Simple sample for title of topic / required'})
-  @Column({ type: DataType.STRING, allowNull: false })
+  @ApiProperty({
+    example: 'NodeJS question',
```
Comment (raman-davidovich) [inline src/topic/topic.model.ts:21]: I propose to leave just a name of the technology ("NodeJS").

File: src/user/dto/create-user.dto.ts [hunk 8]
```diff
@@ -1,12 +1,21 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export class CreateUserDto {
-  @ApiProperty({example: 'Mary', description: 'required name / not uniq / string'})
+  @ApiProperty({
+    example: 'Mary',
+    description: 'required name / not uniq / string',
+  })
   readonly name: string;
```
Comment (raman-davidovich) [inline src/user/dto/create-user.dto.ts:8]: I propose to add surname property as it is in the model.

File: src/user/user.model.ts [hunk 9]
```diff
@@ -1,8 +1,15 @@
...
 import { ApiProperty } from '@nestjs/swagger';
-import { UserAccess } from "./user.access.model";
-import { Quiz } from "../quiz/quiz.model";
-import { AccessGroup } from "../access-group/access-group.model";
+import { UserAccess } from './user.access.model';
+import { Quiz } from '../quiz/quiz.model';
+import { AccessGroup } from '../access-group/access-group.model';
 
 interface UserCreationAttrs {
   name: string;
```
Comment (raman-davidovich) [inline src/user/user.model.ts:15]: I propose to add surname as an optional property.

**Review by gitkoshelew — APPROVED** [review]

**Review by FaridKarmanov — APPROVED** [review]

**Pull request merged**